.github/workflows/ci.yml

```diff
@@ -14,52 +14,27 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node: ["14", "12", engines]
+        node: ["12.22.3", "14.13.1", "16.0.0"]
```

Comment:
I might have missed this in the other PR, but was this required to get the tests passing? If we indeed require 12.22.3 instead of 12.20.0 we need to update `package.json -> engines.node` and the readme as well. Otherwise we can move this back down to `12.20.0`.

Reply:
I think we can revert this back to 12.22.0

Reply:
looks like the test passed.